doc/source/tutorial/optimize.rst

```diff
@@ -1347,6 +1350,171 @@ Preconditioning is an art, science, and industry. Here, we were lucky
+
+By recalling vector multiplication, the objective weights vector :math:`c` of :func:`linprog` in this problem
+should be
+
+.. math::
+        c = [-29, -45, 0, 0]
+
+Next, let's consider the two inequality constraints. These can be converted by using the "greater than" inequality
+constraint to a "less than" inequality constraint by multiplying both sides by a factor of :math:`-1`:
+
+.. math::
+        x_1 -x_2 -3x_3 + 0x_4  &\leq 5\\
+        -2x_1 + 3x_2 + 7x_3 - 3x_4 &\leq -10\\
+
+These equations can be converted as matrix form:
+
+.. math::
+    A_{ub} x \leq b_{ub}\\
+
+where
+
+.. math::
+   :nowrap:
+
+    \begin{equation*} A_{ub} =
+    \begin{bmatrix} 1 & -1 & -3 & 0 \\
+                    -2 & 3 & 7 & -3
+    \end{bmatrix}
+    \end{equation*}
+
+.. math::
+   :nowrap:
+
+    \begin{equation*} b_{ub} =
+    \begin{bmatrix} 5 \\
+                    -10
+    \end{bmatrix}
+    \end{equation*}
+
+Next, let's consider the two equality constraints. These can be converted like:
+
+.. math::
+        2x_1 + 8x_2 + 1x_3 + 0x_4 &= 60\\
+        4x_1 + 4x_2 + 0x_3 + 1x_4 &= 60\\
+
+These equations can be converted as matrix form:
+
+.. math::
+    A_{eq} x = b_{eq}\\
+
+where
+
+.. math::
+   :nowrap:
+
+    \begin{equation*} A_{eq} =
+    \begin{bmatrix} 2 & 8 & 1 & 0 \\
+                    4 & 4 & 0 & 1
+    \end{bmatrix}
+    \end{equation*}
+
+.. math::
+   :nowrap:
+
+    \begin{equation*} b_{eq} =
+    \begin{bmatrix} 60 \\
+                    60
+    \end{bmatrix}
+    \end{equation*}
+
+The last inequality constraint :math:`x_i \geq 0` is minimum value constraint for all decision variables, which can be
+applied as the ``bounds`` argument of :func:`linprog`. As you can see the API doc :func:`linprog`, the default values of
+``bound`` argument is ``(0, None)`` that means all decision variables are non-negative. So, we don't need to do anything
+the last inequality constraint in this case.
+
+Finally, we can solve the desired problem using the mathematical converts and :func:`linprog`.
+
+::
+
+    from scipy.optimize import linprog
+    import numpy as np
+
+    c = np.array([-29.0, -45.0, 0.0, 0.0])
+    A_ub = np.array([[1.0, -1.0, -3.0, 0.0],
+                    [-2.0, 3.0, 7.0, -3.0]])
+    b_ub = np.array([[5.0],
+                    [-10.0]])
+    A_eq = np.array([[2.0, 8.0, 1.0, 0.0],
+                     [4.0, 4.0, 0.0, 1.0]])
+    b_eq = np.array([[60.0],
+                    [60.0]])
+
+    result = linprog(c, A_ub=A_ub, b_ub=b_ub, A_eq=A_eq, b_eq=b_eq)
+
+The result is
+
+::
+
+    >>> print(result)
+    con: array([3.07489501e-09, 3.19440119e-09])
+    fun: -500.79999997382595
+    message: 'Optimization terminated successfully.'
+    nit: 5
+    slack: array([ 1.0000000e+00, -1.0003518e-09])
+    status: 0
+    success: True
+    x: array([9.2000000e+00, 5.2000000e+00, 4.3425461e-11, 2.4000000e+00])
+
+::
+
+    >>> [print("x_", i, ":", np.round(result.x[i], 2)) for i in range(len(result.x))]
```

Comment:
I would suggest using `format` rather than actually rounding.

Reply:
I think this printing function is not needed any more because of the optimization problem change. I delete it.